packages/node/bin/invoke.ts

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env node
```

Comment:
It will and no I'm not using it with `ts-node`. At least not for the container. Yes, we use it like that if you want to run it localy but in production (or in container in our case) you want to run the transpiled version with `node`.